pkg/oci/utils.go

```diff
@@ -39,6 +39,16 @@ var (
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
 )
 
+type CompatOCIProcess struct {
```

Comment:
I don't understand why this is being introduced on a vendoring change? It isn't actively being used so can't (shouldn't?) this be handled on a follow-up PR (along with a new test or two)?